Automate Facebook bot/script.py

```diff
@@ -0,0 +1,35 @@
+import pyautogui
+import time
+
+number = int(input("Enter number of groups: "))
```

Comment:
Instead of asking the number of groups and then taking group ids,
- take the group ids as comma-separated string
- split at comma
- take len of that list (if needed further)